set.mm

```diff
@@ -32381,8 +32394,17 @@ generally appear in a single form (either definitional, but more often
 
     $d x A $.  $d x ph $.  $d x ch $.
     $( Implicit substitution of a class for a setvar variable.  (Contributed by
-       Mario Carneiro, 15-Oct-2016.) $)
+       Mario Carneiro, 15-Oct-2016.)  Avoid ~ ax-10 , ~ ax-11 , ~ ax-12 .
+       (Revised by SN, 2-Sep-2024.) $)
     vtocld $p |- ( ph -> ch ) $=
+      ( cv wceq wcel wex elisset syl wa adantr mpbid exlimddv ) ADJEKZCDAEFLTDM
```

Comment:
Have you had a look at the `bj-vtocl*` series ? They may help, alone or combined, achieve other shortenings or axiom usage reductions.

Reply:
It seems like the last remaining save comes from removing ~ ax-9 by using bj-elisset (well, bj-elissetv also avoids df-clab...)

I'll edit elisset, then (the credit is already correct)